### DISCUSSION TITLE
How can I set Time zone in prisma?

### DISCUSSION THREAD

**Discussion by ilyongHwang** [prisma/prisma#4162] [Q&A] [answered]
When datetime type is inserted, I find an issue in MySQL that is input to **UTC** time.

How can I set Time zone in prisma?

(example in sequelize)
```js
var sequelize = new Sequelize('DB_NAME', 'host', 'pw', {
    host: '127.0.0.1',
    dialect: 'mysql',
    pool: {
        max: 20,
        min: 5, 
        idle: 60000
    },
    // like this
    timezone: '+09:00',
});
```

**Comment by ryands17** [comment, marked as answer]
Hey @ilyongHwang 👋 
That is currently not possible right now. You can follow [this](https://github.com/prisma/prisma/issues/3292) request for more info. As a workaround, the best way would be to convert to the required timezone locally.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
